apps/meet-app/backend/service.bal

```diff
@@ -75,122 +78,357 @@ service http:InterceptableService / on new http:Listener(9090) {
             };
         }
 
-        error? cacheError = userInfoCache.put(userInfo.email, loggedInUser);
-        if cacheError is error {
-            log:printError(string `Error in updating the user cache for: ${userInfo.email}!`);
+        // Fetch the user's privileges based on the roles
+        int[] privileges = [];
+        if authorization:checkPermissions([authorization:authorizedRoles.SALES_TEAM],
+                userInfo.groups) {
```

Comment:
```suggestion
        if authorization:checkPermissions([authorization:authorizedRoles.SALES_TEAM], userInfo.groups) {
```

Reply:
Please check the other locations as well. Wrapping should be done when exceeding 120 characters

Reply:
+1